invoicing_app/tests/tests.py

```diff
@@ -169,12 +174,57 @@ def test_localize_date_old(self):
         my_date = dt(2020, 4, 11, 0, 0)
         self.assertEqual(
             str(self.my_command.localize_date('AR', my_date)),
-            "2020-04-11 00:00:00-03:54"
+            '2020-04-11 00:00:00-03:54'
+        )
+        self.assertEqual(
+            str(self.my_command_new.localize_date('AR', my_date)),
+            '2020-04-11 00:00:00-03:54'
+        )
+
+    def test_today_date_incorrect_format(self):
+        # Not a string in correct format date
+        self.options['today_date'] = 'a.asd-asd??++*asd'
+        with self.assertRaises(Exception):
```

Comment:
Catch the most-specific exception available to you. In this case, I think that'd be `CommandError`. The only problem with that is that it seems to be very generic, and thus I would also assert over the exception message, e.g.
```python
with self.assertRaises(CommandError) as cm:
    do_something()

self.assertEqual(str(cm.exception), 'Expected error message')
```

Reply:
Ok. I'll do it!